src/components/ChangePasswordBox/ChangePasswordBox.css

```diff
@@ -0,0 +1,44 @@
+.changePass-view {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    height: 100vh;
+    background: #222224;
+    width: 100vw;
+}
```

Comment:
We need to make separate class in global file css for this styles because are used in many views